server/api/mixed-content/index.js

```diff
@@ -104,6 +117,18 @@ mixedApi.get('/https-percentage-requests', async (req, res) => {
   let rows = [];
   rows = await queryData(query);
 
+
+  // If there is a certain number of datapoints required for visualization
+  // it will select a subset with proportional skips.
+  const datapoints = req.query.datapoints;
+  if (datapoints) {
+    let mobileRows = rows.filter((row) => row.client === 'mobile');
```

Comment:
nit: Would be better if we use .reduce here to filter both mobile and desktop in only one loop. I think datapoints is the kind of thing that can grow significantly in the future

Reply:
Great, I think we can treat this as a P2 to keep working on what is left and later we try to optimize it.